<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TutorialDialog</class>
 <widget class="QDialog" name="TutorialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="LE_CURRENT">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>30</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_QP_START">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>QP
START</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_QP_STOP">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>QP
STOP</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>539</x>
     <y>389</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>trash</string>
   </property>
   <widget class="QPushButton" name="BTN_SET_MODI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MODI_R4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MODI_R3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MODI_R0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-140</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MODI_L0">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>-140</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_READ_MODI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MODI_L3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>-50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MODI_L1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>-110</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MODI_R2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MODI_L4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>-20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>-170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MODI_R1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-110</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MODI_L2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>-80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_INIT_ALLHAND">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>All
Init</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>-230</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Finger2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_R_GRASP0">
    <property name="geometry">
     <rect>
      <x>-170</x>
      <y>90</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Grasp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_R_RELEASE3">
    <property name="geometry">
     <rect>
      <x>-90</x>
      <y>180</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Release</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Finger2</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_LF_MODE">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Left : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_R_GRASP1">
    <property name="geometry">
     <rect>
      <x>-170</x>
      <y>120</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Grasp</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-220</x>
      <y>30</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-220</x>
      <y>0</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Finger Control</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Finger1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_L_RELEASE1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Release</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_L_RELEASE0">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Release</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_R_RELEASE2">
    <property name="geometry">
     <rect>
      <x>-90</x>
      <y>150</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Release</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Finger4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>each digit means 0.083mA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_L_GRASP0">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Grasp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_R_RELEASE_ALL">
    <property name="geometry">
     <rect>
      <x>-90</x>
      <y>260</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Release</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_R_GRASP2">
    <property name="geometry">
     <rect>
      <x>-170</x>
      <y>150</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Grasp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>All </string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_L_RELEASE4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Release</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_R_GRASP4">
    <property name="geometry">
     <rect>
      <x>-170</x>
      <y>210</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Grasp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_L_GRASP2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Grasp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Finger3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_R_RELEASE1">
    <property name="geometry">
     <rect>
      <x>-90</x>
      <y>120</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Release</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_L_RELEASE2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Release</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>-210</x>
      <y>260</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>All </string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_R_RELEASE0">
    <property name="geometry">
     <rect>
      <x>-90</x>
      <y>90</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Release</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>-230</x>
      <y>210</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Finger4</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_RF_MODE">
    <property name="geometry">
     <rect>
      <x>-230</x>
      <y>40</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Right : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_L_RELEASE_ALL">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Release</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_L_GRASP_ALL">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Grasp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Desired Current (0~100)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_L_GRASP3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Grasp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_R_GRASP_ALL">
    <property name="geometry">
     <rect>
      <x>-170</x>
      <y>260</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Grasp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_R_RELEASE4">
    <property name="geometry">
     <rect>
      <x>-90</x>
      <y>210</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Release</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>-230</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Finger1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_R_GRASP3">
    <property name="geometry">
     <rect>
      <x>-170</x>
      <y>180</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Grasp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>-230</x>
      <y>80</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Thumb
Gripper</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_L_GRASP1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Grasp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_L_GRASP4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Grasp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_L_RELEASE3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Release</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>-230</x>
      <y>180</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Finger3</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="BTN_TEST">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TEST</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_IMU_NULL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Pelvis IMU Nulling / Enable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_INIT_POS">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>INIT
POS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_IMU_ROLL">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>500</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>30</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_IMU_PITCH">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>530</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>30</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_IMU_YAW">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>560</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>30</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Roll</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>560</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Yaw</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>530</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pitch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_TCP_CONNECT">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TCP
CONNECT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
